### DISCUSSION TITLE
AdBlockID Plus kemana?

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#723] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19517680/99477003-34038580-2984-11eb-92b7-8b76aed9a641.png)

bisa dijelaskan ?
apakah anti adblock plus sudah dihilangkan?

**Comment by realodix** [comment, marked as answer]
Sekitar beberapa minggu yang lalu melalui #637, Saya menghapus AdBlockID Plus dan memasukkan semua filter AdBlockID Plus ke dalam AdBlockID.

Jika ada pertanyaan lagi, jangan sungkan..

**Comment by realodix** [comment]
Judul Saya edit, karena yg Saya tangkep issue ini menanyakan AdBlockID Plus, bukan Anti Adblock Plus.

**Comment by 3xploiton3** [comment]
ok, thx infonya, apakah adblockid bisa mendeteksi rule yang tidak pernah digunakan? out of date?
jadi size storage tidak menjadi besarr

**Comment by realodix** [comment]
> ok, thx infonya, apakah adblockid bisa mendeteksi rule yang tidak pernah digunakan? out of date?
> jadi size storage tidak menjadi besarr

Saat ini belum ada cara untuk mendeteksi rule yang usang secara otomatis, hanya bisa dilakukan secara manual. Saya biasanya kalau lagi iseng suka ngecek seperti di https://github.com/realodix/AdBlockID/commit/e323e17c7df465a57e67b1d607b900098ed58a64

Saat ini satu-satunya yang bisa dilakukan secara otomatis adalah dengan mengecek domain yang mati ([dead-hosts/AdBlockID_git_realodix](https://github.com/dead-hosts/AdBlockID_git_realodix)).
